packages/ra-ui-materialui/src/input/InputHelperText.tsx

```diff
@@ -3,7 +3,7 @@ import { isValidElement, ReactElement } from 'react';
 import { useTranslate, ValidationError, ValidationErrorMessage } from 'ra-core';
 
 export const InputHelperText = (props: InputHelperTextProps) => {
-    const { helperText, touched, error } = props;
+    const { helperText, touched = true, error } = props;
```

Comment:
Why do you keep the `touched` prop?

Reply:
To avoid a breaking change (since `InputHelperText` is mentioned in the docs). But if you think that is unnecessary I can remove it.

Reply:
Yes, it's the right moment to clean this up

Reply:
done